src/main/java/uk/gov/gchq/palisade/RequestId.java

```diff
@@ -29,6 +29,7 @@
     private String id;
 
     public RequestId() {
+        //no-args constructor needed.
```

Comment:
I think it would be useful to say why the no arg constructor is needed.

Reply:
Added reason